templates/core/terraform/resource_processor/vmss_porter/locals.tf

```diff
@@ -1,3 +1,9 @@
 locals {
   version = replace(replace(replace(data.local_file.version.content, "__version__ = \"", ""), "\"", ""), "\n", "")
-}
\ No newline at end of file
+}
+locals {
```

Comment:
Why 2 `locals` sections - can they be merge together

Reply:
merged, thanks!